### PR TITLE
feature(integrated-test): add integrated test for tracing-zipkin moudle in agent

### LINKED ISSUE #2739: [Task] Add integrated test for zipkin plugin
### Description

Hi community,
The [tracing-zipkin-plugin](https://shenyu.apache.org/docs/next/user-guide/observability/tracing/#agent-plugin-tracing-zipkin) in `shenyu-agent` has been completed.

And then, we need add integrated test for zipkin plugin.

If you want to take this task, you can comment in this issue.

It might be a little difficult, but you can refer to this [the it of otel plugin](https://github.com/apache/incubator-shenyu/pull/2718)

### Task List

- [ ] Add integrated test for zipkin plugin

Ref #2699 .

### PR DESCRIPTION
**Pull request by lxl910128**
Fixes #2739

### CODE REVIEW

**Comment by JooKS-me** [comment]
Thanks~

File: shenyu-integrated-test/shenyu-integrated-test-agent/shenyu-integrated-test-agent-tracing/shenyu-integrated-test-agent-tracing-zipkin/src/test/java/org/apache/shenyu/integrated/test/agent/zipkin/ZipkinPluginTest.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+    private static final Gson GSON = new Gson();
+
+    @BeforeClass
+    public static void setup() throws IOException, InterruptedException {
+        OrderDTO user = new OrderDTO("123", "Tom");
+        user = HttpHelper.INSTANCE.postGateway("/http/order/save", user, OrderDTO.class);
+        assertEquals("hello world save order", user.getName());
+        // Sleep for jaeger to react
+        Thread.sleep(5000);
```
Comment (JooKS-me) [inline shenyu-integrated-test/shenyu-integrated-test-agent/shenyu-integrated-test-agent-tracing/shenyu-integrated-test-agent-tracing-zipkin/src/test/java/org/apache/shenyu/integrated/test/agent/zipkin/ZipkinPluginTest.java:61]: Do we need sleep here?

Comment (JooKS-me) [reply shenyu-integrated-test/shenyu-integrated-test-agent/shenyu-integrated-test-agent-tracing/shenyu-integrated-test-agent-tracing-zipkin/src/test/java/org/apache/shenyu/integrated/test/agent/zipkin/ZipkinPluginTest.java:61]: Sorry, please ignore this comment.

File: shenyu-integrated-test/shenyu-integrated-test-agent/shenyu-integrated-test-agent-tracing/shenyu-integrated-test-agent-tracing-zipkin/src/test/java/org/apache/shenyu/integrated/test/agent/zipkin/ZipkinPluginTest.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    private static final String ZIPKIN_HTTP_SERVER = "http://localhost:9411/api/v2/traces?serviceName=shenyu-agent";
+
+    private static final Gson GSON = new Gson();
+
+    @BeforeClass
+    public static void setup() throws IOException, InterruptedException {
+        OrderDTO user = new OrderDTO("123", "Tom");
+        user = HttpHelper.INSTANCE.postGateway("/http/order/save", user, OrderDTO.class);
+        assertEquals("hello world save order", user.getName());
+        // Sleep for jaeger to react
```
Comment (JooKS-me) [inline shenyu-integrated-test/shenyu-integrated-test-agent/shenyu-integrated-test-agent-tracing/shenyu-integrated-test-agent-tracing-zipkin/src/test/java/org/apache/shenyu/integrated/test/agent/zipkin/ZipkinPluginTest.java:60]: Pls change the `jaeger` to `zipkin`~

**Review by JooKS-me — CHANGES_REQUESTED** [review]

File: shenyu-integrated-test/shenyu-integrated-test-agent/shenyu-integrated-test-agent-tracing/shenyu-integrated-test-agent-tracing-zipkin/src/test/java/org/apache/shenyu/integrated/test/agent/zipkin/ZipkinPluginTest.java (hunk 2 above)
Comment (lxl910128) [reply shenyu-integrated-test/shenyu-integrated-test-agent/shenyu-integrated-test-agent-tracing/shenyu-integrated-test-agent-tracing-zipkin/src/test/java/org/apache/shenyu/integrated/test/agent/zipkin/ZipkinPluginTest.java]: done

**Review by JooKS-me — APPROVED** [review]

**Pull request merged**